ddf-rpc-api/pom.xml

```diff
@@ -0,0 +1,292 @@
+                            org.codice.ddf.configuration;version="[2.19,3)",
+
+                            org.apache.commons.lang3.tuple;version="[3.8.0,4.0.0)",
+                            org.apache.commons.lang3;version="[3.8.0,4.0.0)",
+
+                            org.apache.shiro.subject;version="[1.4,2)",
+                            org.apache.shiro;version="[1.4.0,2.0.0)",
+
+                            org.opengis.annotation;version="[19.1.0,21.0.0)",
+                            org.opengis.filter.expression;version="[19.1.0,21.0.0)",
+                            org.opengis.filter.sort;version="[19.1.0,21.0.0)",
+                            org.opengis.filter;version="[19.1.0,21.0.0)",
+                            *
+                        </Import-Package>
+                    </instructions>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>com.coveo</groupId>
+                <artifactId>fmt-maven-plugin</artifactId>
+                <version>2.3.0</version>
+                <executions>
+                    <execution>
+                        <phase>validate</phase>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-release-plugin</artifactId>
+                <version>${maven.release.plugin.version}</version>
+                <configuration>
+                    <autoVersionSubmodules>true</autoVersionSubmodules>
+                    <preparationGoals>clean verify install</preparationGoals>
+                    <pushChanges>false</pushChanges>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>0.8.2</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.codice.maven</groupId>
+                        <artifactId>jacoco</artifactId>
+                        <version>${codice-maven.version}</version>
+                    </dependency>
+                </dependencies>
+                <executions>
+                    <execution>
+                        <id>default-prepare-agent</id>
+                        <goals>
+                            <goal>prepare-agent</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>default-report</id>
+                        <phase>prepare-package</phase>
+                        <configuration>
+                            <outputDirectory>
+                                ${project.build.directory}/site/${project.report.output.directory}/jacoco/
```

Comment:
✏️ I think `${project.report.output.directory}` is a custom property so make sure it's defined somewhere

Reply:
I removed for now since i'm not sure what it was doing for us anyways.